src/service/viewer-impl.js

```diff
@@ -109,7 +109,7 @@ export const VisibilityState = {
  * @export {!Array<!RegExp>}
  */
 export const TRUSTED_VIEWER_HOSTS = [
-  /^(.*\.)?(google)(\.com?)?(\.[a-z]{2})?$/,
+  /(^|\.)google\.(com?|[a-z]{2}|com?\.[a-z]{2})$/,
```

Comment:
No, but they just complicate the regex and [don't make](https://jsperf.com/regex-capture-vs-non-capture) it faster.